lib/edamam_wrapper.rb

```diff
@@ -0,0 +1,41 @@
+require 'httparty'
+
+class EdamamWrapper
+  class EdamamError < StandardError; end
+
+  BASE_URL = "https://api.edamam.com/search"
+  ID = ENV["MUNCHER_ID"]
+  KEY = ENV["MUNCHER_KEYS"]
+
+  def self.search_recipes(query)
+    url = BASE_URL + "?q=#{query}" + "&app_id=#{ID}" + "&app_key=#{KEY}" + "&from=0" + "&to=50"
+    response = HTTParty.get(url)
+    raise_on_error(response)
```

Comment:
If you're going to raise an error here, your controller would need to `rescue` the error.